prow/spyglass/artifacts.go

```diff
@@ -0,0 +1,126 @@
+package spyglass
```

Comment:
Nothing in this package is new; I just moved it out of `spyglass.go` to make things slightly easier to read.